src/txn.h

```diff
@@ -39,9 +39,8 @@ typedef enum {
 typedef struct {
     txnElemType_e elemType; // type of most-recently-seen element
 
-    uint8_t outVal[128];    // most-recently-seen currency value, in decimal
-    uint8_t valLen;         // length of outVal
-    uint8_t outAddr[77];    // most-recently-seen address
+    uint8_t outVal[24];    // most-recently-seen currency value, Sia-encoded
+    uint8_t outAddr[32];    // most-recently-seen address, Sia-encoded
```

Comment:
since we're no longer streaming, I suppose the "most-recently-seen" comment can be dropped